composer.json

```diff
@@ -1,23 +1,30 @@
 {
     "name": "laminas/laminas-dependency-plugin",
-    "description": "Replace zendframework and zfcampus packages with their Laminas Project equivalents.",
     "type": "composer-plugin",
+    "description": "Replace zendframework and zfcampus packages with their Laminas Project equivalents.",
     "license": "BSD-3-Clause",
-    "config": {
-        "sort-packages": true
-    },
```

Comment:
please revert all unrelated changes in that file

Reply:
reverted